tests/platform_tests/test_advanced_reboot.py

```diff
@@ -77,6 +78,20 @@ def pytest_generate_tests(metafunc):
         metafunc.parametrize("sad_case_type", sad_cases, scope="module")
 
 
+@pytest.fixture(scope="function")
+def skip_on_low_cpu_platform(duthosts, rand_one_dut_hostname, nbrhosts):
+    '''
+    skip the test on some low CPU platform if the neighbor is not vsonic
+    '''
+    LOW_CPU_PLATFORMS = ["x86_64-mlnx_msn2700-r0"]
+    duthost = duthosts[rand_one_dut_hostname]
+    platform = duthost.facts["platform"]
+    if platform in LOW_CPU_PLATFORMS:
+        for nbrhost in list(nbrhosts.values()):
+            if not isinstance(nbrhost, SonicHost):
```

Comment:
Might be better to use `if request.config.getoption("neighbor_type") != "sonic":` instead.

Reply:
Yeah, it works. Thanks!